.releaserc

```diff
@@ -1,8 +1,21 @@
 {
   "tagFormat": "${version}",
+  "branches": [
+    { "name": "master", "prerelease": false },
+    { "name": "development", "prerelease": "staging" }
+  ],
   "plugins": [
     [
-      "@semantic-release/commit-analyzer"
+      "@semantic-release/commit-analyzer",{
+        "preset": "angular",
+        "releaseRule": [
```

Comment:
you can remove All of these release rules as they are already part of the angular preset